img/logo_vector_footer.svg

```diff
@@ -0,0 +1,11 @@
+<path d="M93.5569 15.5377C92.7246 15.5377 91.9489 15.4237 91.2295 15.1957C90.5242 14.9535 89.9036 14.5973 89.3676 14.1271C88.8457 13.6569 88.4296 13.0656 88.1193 12.3532C87.8231 11.6408 87.675 10.8073 87.675 9.85264C87.675 8.91226 87.8231 8.10724 88.1193 7.43757C88.4296 6.75366 88.8316 6.19798 89.3253 5.77053C89.819 5.32883 90.3832 5.00825 91.0179 4.80877C91.6526 4.59505 92.3015 4.48819 92.9644 4.48819C93.712 4.48819 94.3891 4.60217 94.9956 4.83015C95.6162 5.05812 96.1452 5.37158 96.5824 5.77053C97.0338 6.16948 97.3794 6.64679 97.6192 7.20247C97.873 7.75816 98 8.3637 98 9.01912C98 9.50356 97.866 9.87402 97.598 10.1305C97.33 10.387 96.9562 10.5508 96.4766 10.622L91.2506 11.4128C91.4058 11.883 91.7232 12.2392 92.2028 12.4814C92.6823 12.7094 93.2324 12.8234 93.8531 12.8234C94.4314 12.8234 94.9744 12.7522 95.4822 12.6097C96.0041 12.4529 96.4273 12.2748 96.7517 12.0754C96.9774 12.2178 97.1678 12.4173 97.3229 12.6738C97.4781 12.9303 97.5557 13.201 97.5557 13.4859C97.5557 14.1271 97.2595 14.6044 96.6671 14.9179C96.2157 15.1601 95.7079 15.324 95.1437 15.4094C94.5795 15.4949 94.0505 15.5377 93.5569 15.5377ZM92.9644 7.13836C92.6259 7.13836 92.3297 7.19535 92.0758 7.30934C91.836 7.42332 91.6385 7.57293 91.4834 7.75816C91.3282 7.92913 91.2083 8.12861 91.1237 8.35658C91.0532 8.5703 91.0109 8.79115 90.9968 9.01912L94.6147 8.4207C94.5724 8.13573 94.4173 7.85077 94.1493 7.5658C93.8813 7.28084 93.4863 7.13836 92.9644 7.13836Z" fill="white"/>
+</svg>
```

Comment:
You can use **one** svg-image for a logo in your footer and header, just replace color in fill attribute
`fill="currentColor"` or use this in your css
More info 
[link1](https://css-tricks.com/cascading-svg-fill-color/) 
[link2](https://gomakethings.com/currentcolor-and-svgs/)